docs/contributing.md

```diff
@@ -36,3 +36,18 @@ If you need ideas, you can check out the [help wanted](https://github.com/jxnl/i
 <a href="https://github.com/jxnl/instructor/graphs/contributors">
   <img src="https://contrib.rocks/image?repo=jxnl/instructor" />
 </a>
+
+
+## Additional Resources
```

Comment:
can you mvoe this above the contributors table and something  like 

to run the docs pip install requirements-doc.txt and run `mkdocs serve -w .` to preview the documentation

to improve the legibility of code and content consider using 


hl_lines... ...
admonitinos: ...

and include a code example in our docs so we dont /have/ to jump links ( can literally copy paste a snippet of their markdown)

Reply:
Already make the requested changes and also leave in "Extended description
"  a second version of the corrections. Regarding (can literally copy paste a snippet of their markdown) if you can guide me how, I'll do it asap. thanks!